back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/config/WebSecurityConfig.java

```diff
@@ -65,6 +65,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 "/webjars/**").permitAll()
             .antMatchers("/join").permitAll()
             .antMatchers("/login").permitAll()
+            .antMatchers("/mail-auth").permitAll()
```

Comment:
하나로 합체해도 될 것 같긴 한데... 취향 차이일지도 모르겠네요~!

```suggestion
                "/webjars/**",
                "/join",
                "/login",
                "/mail-auth").permitAll()
```

Reply:
일괄되도록 수정했습니다~